### PR TITLE
Use CUDA 11.2 for nightly docker build.

### PR DESCRIPTION
**Pull request by xuzhao9**
Set CUDA_VERSION to 11.2.0 since Nvidia name their docker image on Ubuntu 18.04 to be nvidia/cuda:11.2.0-cudnn8-devel-ubuntu18.04.

Note that cudatoolkit 11.2.0 is not yet on [conda](https://repo.anaconda.com/pkgs/main/linux-64/), and we need to wait for that before merging this PR.

- https://hub.docker.com/r/nvidia/cuda/

### CODE REVIEW

**Comment by seemethere** [comment]
You'll need to use the `-c conda-forge` flag to get `cudatoolkit=11.2`

**Comment by xuzhao9** [comment]
> You'll need to use the `-c conda-forge` flag to get `cudatoolkit=11.2`

Thanks! I added the `CUDA_VERSION` argument to Dockerfile and docker.Makefile to install cudatoolkit from conda-forge channel.

File: Dockerfile [hunk 1]
```diff
@@ -52,9 +52,10 @@ RUN --mount=type=cache,target=/opt/ccache \
 FROM conda as conda-installs
 ARG PYTHON_VERSION=3.8
 ARG CUDA_VERSION=11.0
+ARG CUDA_CHANNEL=anaconda
```
Comment (seemethere) [inline Dockerfile:55]: Shouldn't this be `defaults`?

I haven't actually tried using the `anaconda` channel before

Comment (xuzhao9) [reply Dockerfile:55]: I found the `anaconda` channel in the Anaconda Cloud: https://anaconda.org/anaconda/cudatoolkit. But `conda config --show channels` gives `defaults`. I have updated to use the `defaults` channel.

**Review by seemethere — APPROVED** [review]

**Pull request closed without merging**